src/main/java/sixgaezzang/sidepeek/skill/domain/Skill.java

```diff
@@ -16,14 +19,25 @@
 @Getter
 public class Skill {
 
+    public static final int MAX_SKILL_NAME_LENGTH = 50;
+
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @Column(name = "name", nullable = false, length = 50)
+    @Column(name = "name", nullable = false, length = MAX_SKILL_NAME_LENGTH)
     private String name;
 
     @Column(name = "icon_image_url", nullable = false, columnDefinition = "TEXT")
     private String iconImageUrl;
 
+    @Builder
+    public Skill(String name, String iconImageUrl) {
+        validateMaxLength(name, MAX_SKILL_NAME_LENGTH,
+            "최대 " + MAX_SKILL_NAME_LENGTH + "자의 이름으로 생성할 수 있습니다.");
```

Comment:
url형식도 validate하는건 어떠세욤?
```suggestion
        validateMaxLength(name, MAX_SKILL_NAME_LENGTH,
            "최대 " + MAX_SKILL_NAME_LENGTH + "자의 이름으로 생성할 수 있습니다.");
            validateURI(iconImageUrl, "이미지 url 형식이 올바르지 않습니다.");
```

Reply:
저도 S3에서 넘어오는 URL 형식이 정해져있다면, validate 하는 거 찬성합니다!

Reply:
파일 이름은 uuid로 생성하고 있는데 정규식 보니까 적용해도 문제는 없을 것 같은데 적용해보겠습니다!!!!